### DISCUSSION TITLE
Allow folders in Prometheus Alerts [unified alerting]

### DISCUSSION THREAD

**Discussion by LawrenceWarren** [grafana/grafana#58744] [Alerting]
Alerts which are written directly into Grafana can be added into folders via the GUI. They display neatly.
Alerts which are written in Prometheus and are bought into the Grafana GUI as part of the Unified Alerting system are displayed in folders which have the name of the files they are stored in. As such, they are not very well organised (or pretty), as shown below.

<img width="1410" alt="image" src="https://user-images.githubusercontent.com/53704822/201881704-660d7d3e-b6af-4bd3-a427-7e13d7e971da.png">

Is there a way to organise unified alerts into folders, e.g. via some kind of label? If not, can this be noted as a feature request for a future release?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
